tests/test_utils.py

```diff
@@ -7,26 +7,45 @@
 import mock
 
 from importanize.utils import (
-    ignore_site_packages_paths,
+    site_packages_paths,
     is_std_lib,
+    is_site_packages,
     list_strip,
     read,
 )
 
 
 class TestUtils(unittest.TestCase):
-    def test_ignore_site_packages_paths(self):
+    def test_site_packages_paths(self):
         sys.path.append(os.getcwd())
         paths = sys.path[:]
 
-        with ignore_site_packages_paths():
+        with site_packages_paths(lambda i: "site-packages" not in i):
             self.assertNotEqual(sys.path, paths)
             self.assertLess(len(sys.path), len(paths))
 
         self.assertIn(os.getcwd(), sys.path)
         self.assertListEqual(sys.path, paths)
         sys.path.remove(os.getcwd())
 
+    def test_site_packages_paths_exception(self):
```

Comment:
up to you 😄 

I usually like to leave old tests so that I have some confidence new refactor does not break old stuff.